src/pages/dev/send-tokens/interchain-tokens/upgrade-tokens.mdx

```diff
@@ -4,115 +4,33 @@ import { Callout } from "/src/components/callout";
-- Transfer Interchain Tokens between chains via the Interchain Token Service by calling the [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/interfaces/IInterchainTokenService.sol#L210)
-
-You can optionally have any of these custom tokens extend [`IInterchainTokenStandard`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainTokenStandard.sol) to offer [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/a2dfcb2490497e627b66be789d944ec3260c5eea/contracts/interfaces/IInterchainTokenStandard.sol#L20) and [`interchainTransferFrom`](https://github.com/axelarnetwork/interchain-token-service/blob/a2dfcb2490497e627b66be789d944ec3260c5eea/contracts/interfaces/IInterchainTokenStandard.sol#L37) methods directly on your token.
-
-> You could decide to make all of these Token Managers [Mint/Burn](https://docs.axelar.dev/dev/glossary#mintburn), or you could specify at most one (likely your primary/origin chain) to be [Lock/Release](https://docs.axelar.dev/dev/glossary#lockunlock) as indicated in the [`ITokenManagerType.sol`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol) depending on your needs.
-
-Here is an example of deploying a token manager:
-
-```solidity
-/**
- * @notice Used to deploy TokenManagers.
- * @dev At least the `gasValue` amount of native token must be passed to the
- * function call. `gasValue` exists because this function can be part of a
- * multicall involving multiple functions that could make remote contract calls.
- * @param salt The salt to be used during deployment.
- * @param destinationChain The name of the chain to deploy the TokenManager and
- * standardized token to.
- * @param tokenManagerType The type of TokenManager to be deployed.
- * @param params The params that will be used to initialize the TokenManager.
- * @param gasValue The amount of native tokens to be used to pay for gas for the
- * remote deployment.
- * @return tokenId The tokenId corresponding to the deployed TokenManager.
- */
-function deployTokenManager(
-    bytes32 salt,
-    string calldata destinationChain,
-    TokenManagerType tokenManagerType,
-    bytes calldata params,
-    uint256 gasValue
-) external payable
-```
+If you have a token deployed on multiple chains and want to make it an Interchain Token:
 
-To create the `params` that will be used to initialize the Token Manager in the `deployTokenManager` method above, you can use the [Online Ethereum ABI Encoder](https://adibas03.github.io/online-ethereum-abi-encoder-decoder/#/encode) specifying the operator of the Token Manager and the token address with the argument type `bytes` and `address` respectively.
```

Comment:
This paragraph is very important to help builders understand how to create `params` for deploying token managers.

This line `The [Online Ethereum ABI Encoder](https://adibas03.github.io/online-ethereum-abi-encoder-decoder/#/encode) can create the parameters for initializing the token manager.` is confusing because, on the smart contract, we referenced it as `params` not parameters that sound like the entire parameters meanwhile the online encoder is just used to create `params.`